tests/specs/lint/stdin_json/expected_from_stdin_json.out

```diff
@@ -20,6 +20,7 @@
       "hint": [WILDCARD]
     }
   ],
+  "skipped": [],
```

Comment:
I'm not sure about adding this to the json output for now. What if we find we don't want to do this in the future?

Maybe we should exclude "skipped" for the time being?